app-backend/api/src/main/scala/exports/QueryParameters.scala

```diff
@@ -0,0 +1,17 @@
+package com.azavea.rf.api.exports
+
+import com.azavea.rf.database.query._
+import com.azavea.rf.api.utils.queryparams._
+
+import akka.http.scaladsl.server.Directives._
+import akka.http.scaladsl.server.directives.ParameterDirectives.parameters
+
+import java.util.UUID
+
+trait ExportQueryParameterDirective extends QueryParametersCommon {
+  val exportQueryParams = parameters((
+    'organization.as[UUID].?,
```

Comment:
I think this is to filter by a particular organization when hitting the API, e.g. limit exports to organization ABC would be /api/exports/?organizaton=ABC